.env

```diff
@@ -1 +1,6 @@
+PG_USER = kimenyi
+PG_HOST = localhost
+PG_DATABASE = FreeMentor
+PG_PASSWORD = kigali
```

Comment:
You aren't supposed to commit the `.env` file to git, it has sensitive info. Instead, create a .env.sample file that looks like the `.env` file but replace the actual data with dummy data.

Please remove this file from git and replace it with the `.env.sample` file